opsdroid/connector/telegram/__init__.py

```diff
@@ -182,7 +181,7 @@ def build_url(self, method):
             ):
                 self.latest_update = result["update_id"] + 1
                 _LOGGER.debug(
-                    _("Emoji message parsing not supported " "- Ignoring message")
+                    _("Emoji message parsing not supported " "- Ignoring message.")
```

Comment:
Logging should be together:

```suggestion
                    _("Emoji message parsing not supported - Ignoring message.")
```